src/components/voucher/VoucherCollectionTabs.tsx

```diff
@@ -16,13 +16,24 @@ const VoucherCollectionTabs: React.VFC<{ vouchers: VoucherProps[] }> = ({ vouche
   }[] = [
     {
       key: 'available',
-      name: formatMessage(commonMessages.status.available),
-      isDisplay: voucher => !!voucher.available,
+      name: formatMessage(usersMessages.tab.available),
+      isDisplay: voucher => voucher.available,
+    },
+    {
+      key: 'notYet',
+      name: formatMessage(usersMessages.tab.notYet),
+      isDisplay: voucher =>
+        (voucher.startedAt && voucher.startedAt.getTime() > Date.now() && !voucher.status.used) || false,
     },
     {
       key: 'unavailable',
-      name: formatMessage(commonMessages.status.expired),
-      isDisplay: voucher => !voucher.available,
+      name: formatMessage(usersMessages.tab.expired),
+      isDisplay: voucher => (voucher.endedAt && voucher.endedAt.getTime() < Date.now()) || false,
```

Comment:
NITS: equivalent to `voucher.status.outdated` ?

Reply:
not equl

sql from voucher_status

```
SELECT
  DISTINCT voucher.id AS voucher_id,
  (
    (
      (voucher_plan.started_at IS NOT NULL)
      AND (voucher_plan.started_at > now())
    )
    OR (
      (voucher_plan.ended_at IS NOT NULL)
      AND (voucher_plan.ended_at < now())
    )
  ) AS outdated,
  (t.order_id IS NOT NULL) AS used
```